closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -365,6 +368,18 @@ static jlong extractVersionToJlong(const char *astring)
 }
 
 static void *crypto_library = NULL;
+
+/*
+ * Class:     jdk_crypto_jniprovider_NativeCrypto
+ * Method:    isOpenSSLFIPS
+ * Signature: ()Z
+ */
+JNIEXPORT jboolean JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_isOpenSSLFIPS
+  (JNIEnv *env, jclass thisObj)
```

Comment:
`thisObj` is a misleading name, the parameter is a class:
```c
  (JNIEnv *env, jclass clazz)
```

Reply:
Changed parameter name.